src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -642,17 +642,22 @@ public ResponseEntity<String> getShortCircuitResult(@Parameter(description = "st
                 ResponseEntity.noContent().build();
     }
 
-    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/shortcircuit/results/fault_results/paged")
-    @Operation(summary = "Get a fault results page for the short circuit analysis result on study")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The short circuit analysis result fault results page"),
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/shortcircuit/results/paged")
+    @Operation(summary = "Get a short circuit analysis results page on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The short circuit analysis results page"),
         @ApiResponse(responseCode = "204", description = "No short circuit analysis has been done yet"),
         @ApiResponse(responseCode = "404", description = "The short circuit analysis has not been found")})
-    public ResponseEntity<String> getShortCircuitAnalysisFaultResultsPage(@Parameter(description = "study UUID") @PathVariable("studyUuid") UUID studyUuid,
-                                                               @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid,
-                                                               @Parameter(description = "Full or only those with limit violations or none fault results") @RequestParam(name = "mode", required = false, defaultValue = "WITH_LIMIT_VIOLATIONS") String mode,
-                                                               Pageable pageable) {
-        String faultResultsPage = shortCircuitService.getShortCircuitAnalysisFaultResultsPage(nodeUuid, mode, pageable);
-        return faultResultsPage != null ? ResponseEntity.ok().body(faultResultsPage) :
+    public ResponseEntity<String> getShortCircuitAnalysisResultsPage(@Parameter(description = "study UUID") @PathVariable("studyUuid") UUID studyUuid,
```

Comment:
I'm not sure about this, should I make 2 endpoints ?

Reply:
I would have made 2 endpoints yes, one for faultResultPage and one for FeederResultPage yes.

Reply:
It have been decided to remove this endpoint and use only the legacy one, as for sensi